srcpkgs/freerdp3/template

```diff
@@ -0,0 +1,143 @@
+ libxkbfile-devel mit-krb5-devel openssl-devel pcsclite-devel
+ pkcs11-helper-devel pulseaudio-devel uriparser-devel wayland-devel"
+short_desc="Free RDP (Remote Desktop Protocol) client"
+maintainer="chrysos349 <chrysostom349@gmail.com>"
+license="Apache-2.0"
+homepage="https://www.freerdp.com/"
+changelog="https://raw.githubusercontent.com/FreeRDP/FreeRDP/3.5.0/ChangeLog"
+distfiles="https://github.com/FreeRDP/FreeRDP/archive/${version}.tar.gz"
+checksum=03323b383980ee91decbed88270bac061ffb17fd04e52576c70da7885601ecbe
+CFLAGS="-Wno-dev"
+
+# help cmake generate manuals when cross-compiling
+post_patch() {
+	vsed -i cmake/InstallFreeRDPMan.cmake \
+		-e '/generate_argument_docbook/s/COMMAND/& \${CMAKE_CROSSCOMPILING_EMULATOR}/'
+}
+
+pre_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		export CXX="$XBPS_WRAPPERDIR/${XBPS_CROSS_TRIPLET}-c++"
+		configure_args+="
+		 -DCMAKE_CXX_COMPILER_AR=/usr/bin/${XBPS_CROSS_TRIPLET}-ar
+		 -DCMAKE_CXX_COMPILER_RANLIB=/usr/bin/${XBPS_CROSS_TRIPLET}-ranlib"
+	fi
+}
+
+case "$XBPS_TARGET_MACHINE" in
+	i686*|x86_64*) configure_args+=" -DWITH_SSE2=ON";;
+	armv5tel*) configure_args+=" -DWITH_NEON=OFF";;
+esac
+
+# first we want to separate -server, everything else then goes to client
+subpackages="libfreerdp-server3 freerdp-server3 freerdp-server3-devel
+ libfreerdp3 freerdp3-devel"
+
+libfreerdp3_package() {
+	short_desc+=" - runtime libraries"
+	pkg_install() {
+		vmove "usr/lib/*.so.*"
+	}
+}
+
+freerdp3-devel_package() {
+	short_desc+=" - development files"
+	depends="cJSON-devel libfreerdp3>=${version}_${revision}
+	 openssl-devel"
+	pkg_install() {
+		vmove "usr/lib/*.so"
+		vmove usr/include
+		vmove usr/lib/cmake
+		vmove usr/lib/pkgconfig
+	}
+}
+
+libfreerdp-server3_package() {
+	short_desc="Free RDP (Remote Desktop Protocol) server - runtime files"
+	depends="libfreerdp3>=${version}_${revision}"
+	pkg_install() {
+		vmove "usr/lib/freerdp3/server/proxy/plugins"
+		vmove "usr/lib/libfreerdp-server*.so.*"
+		vmove "usr/lib/libfreerdp-shadow*.so.*"
+	}
+}
+
+freerdp-server3_package() {
+	short_desc="Free RDP (Remote Desktop Protocol) server"
+	depends="libfreerdp-server3>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/bin/freerdp-proxy3
+		vmove usr/bin/freerdp-shadow-cli3
+		vmove usr/share/man/man1/freerdp-proxy3.1
+		vmove usr/share/man/man1/freerdp-shadow-cli3.1
+	}
+}
+
+freerdp-server3-devel_package() {
+	depends="libfreerdp-server3>=${version}_${revision}
+	 freerdp3-devel>=${version}_${revision}"
+	short_desc="Free RDP (Remote Desktop Protocol) server - development files"
+	pkg_install() {
+		vmove "usr/lib/libfreerdp-server*.so"
+		vmove "usr/lib/libfreerdp-shadow*.so"
+		vmove usr/lib/cmake/FreeRDP-Proxy3
+		vmove usr/lib/cmake/FreeRDP-Server3
+		vmove usr/lib/cmake/FreeRDP-Shadow3
+		vmove usr/lib/pkgconfig/freerdp-server-proxy3.pc
+		vmove usr/lib/pkgconfig/freerdp-server3.pc
+		vmove usr/lib/pkgconfig/freerdp-shadow3.pc
+	}
+}
```

Comment:
Just lump this in with `freerdp3-devel`; there isn't much value in separating server development files.